### PR TITLE
feat: 中华网新增RSS

### PR DESCRIPTION
**Pull request by jiaaoMario**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/zhonghuawang/news/:category?
/zhonghuawang/news/military
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note
新增中华网军事，新闻时事 等路由，后续会补充更多中华网相关内容，如财经新闻等～

### CODE REVIEW

File: lib/v2/zhonghuawang/radar.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    'military.china.com': {
+        _name: '中华网军事',
+        '.': [
```
Comment (TonyRL) [inline lib/v2/zhonghuawang/radar.js:4]:
```suggestion
    'china.com': {
        _name: '中华网',
        'military': [
```

File: lib/v2/zhonghuawang/radar.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            {
+                title: '军事新闻',
+                docs: 'https://docs.rsshub.app/new-media.html#zhong-hua-wang',
+                source: '/news',
+                target: '/news/military',
```
Comment (TonyRL) [inline lib/v2/zhonghuawang/radar.js:9]:
```suggestion
                target: '/zhonghuawang/news/military',
```

File: lib/v2/zhonghuawang/radar.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        ],
+    },
+    'news.china.com': {
+        _name: '中华网',
+        '.': [
+            {
+                title: '时事新闻',
+                docs: 'https://docs.rsshub.app/new-media.html#zhong-hua-wang',
+                source: '/:category',
+                target: '/news/:category',
```
Comment (TonyRL) [inline lib/v2/zhonghuawang/radar.js:20]:
```suggestion
                target: '/zhonghuawang/news/:category',
```

File: lib/v2/zhonghuawang/maintainer.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    '/news/military': ['jiaaoMario'],
+    '/news': ['jiaaoMario'],
```
Comment (TonyRL) [inline lib/v2/zhonghuawang/maintainer.js:3]: This should match with `router.js`
```suggestion
    '/news/:category?': ['jiaaoMario'],
```

File: docs/en/new-media.md [hunk 5]
```diff
@@ -131,6 +131,25 @@ Compared to the official one, the RSS feed generated by RSSHub not only has more
 
 </RouteEn>
 
+## China.com
+
+### Military - Military News
+
+<RouteEn author="jiaaoMario" example="/zhonghuawang/military/news/hot" path="/zhonghuawang/military/news/hot">
```
Comment (TonyRL) [inline docs/en/new-media.md:138]: `/zhonghuawang/military/news/hot` does not work

File: docs/new-media.md [hunk 6]
```diff
@@ -4214,6 +4214,25 @@ wechat-feeds 来源[已停止更新](https://github.com/hellodword/wechat-feeds/
 
 <Route author="nczitzk" example="/cria/news/1" path="/cria/news/:id?" :paramsDesc="['列表 id，可在列表页的 URL 中找到，默认为首页']"/>
 
+## 中华网
+
+### 军事 - 军事新闻
+
+<Route author="jiaaoMario" example="/zhonghuawang/military/news/hot" path="/zhonghuawang/military/news/hot">
```
Comment (TonyRL) [inline docs/new-media.md:4221]: `/zhonghuawang/military/news/hot` does not work

**Comment by jiaaoMario** [comment]
@TonyRL Thx for you cr. I have solved it.  Looking forward to your reply

**Pull request closed without merging**